doc/changelog/03-notations/12965-master+fix9569-propagage-binding-vars-notations.rst

```diff
@@ -0,0 +1,6 @@
+- **Fixed:**
+  Occasional unexpected capture of the name of global references by
```

Comment:
```suggestion
  Capture of the name of global references by
```
IMO all bugs are unexpected and don't manifest themselves always ;-)

Reply:
OK